.meta/sinks/gcp_cloud_storage.toml.erb

```diff
@@ -148,12 +148,12 @@ description = "The filename extension to use in the object name."
 [sinks.gcp_cloud_storage.options.compression]
 type = "string"
 common = true
-required = true
+default = "none"
 description = "The compression mechanism to use."
 
 [sinks.gcp_cloud_storage.options.compression.enum]
-gzip = "GZIP compression"
 none = "No compression"
+gzip = "GZIP compression"
```

Comment:
I know it was all-caps before, but "gzip" should probably not be all-caps as it is not an acronym.